addons/website_slides_survey/controllers/slides.py

```diff
@@ -49,6 +51,14 @@ def _get_slide_detail(self, slide):
 
         return result
 
+    # Utils
+    # ---------------------------------------------------
+    def _slide_set_completed_check(self, slide):
+        result = super(WebsiteSlides, self)._slide_set_completed_check(slide)
```

Comment:
Related to previous comment, you could override `_set_completed_slide` here, check slide_type, raise if certification else return super.